TROUBLESHOOTING.md

```diff
@@ -0,0 +1,106 @@
+# Troubleshooting
+
+This is a list of common issues along with troubleshooting tips.
+
+If filing a Github issue, please include:
+* Information about your environment (OS, GPU, ...).
+* Troubleshooting steps that you have already attempted.
+* Screenshots, if applicable.
+
+- [Common Issues](#common-issues)
+  - [Habitat-Lab interactive\_play.py](#habitat-lab-interactive_playpy)
+  - [Unable to create context](#unable-to-create-context)
+  - [Black squares on NVIDIA A100](#black-squares-on-nvidia-a100)
+- [Graphics Troubleshooting Tips](#graphics-troubleshooting-tips)
+    - [General Tips](#general-tips)
+    - [Linux](#linux)
+    - [Windows](#windows)
+
+
+## Common Issues
+
+### Habitat-Lab interactive_play.py
+
+On some systems, `examples/interactive_play.py` crashes due to the following error:
+
+```
+X Error of failed request:  BadAccess (attempt to access private resource denied)
+```
+
+This is an ongoing issue related to how the underlying `pygame` library interacts with Habitat. A replacement for the `interactive_play` script is [on the roadmap](habitat-hitl/README.md).
+
+### Unable to create context
+
+Driver or graphics configuration issues commonly manifest as such:
+
+```
+Platform::WindowlessEglApplication::tryCreateContext(): unable to find CUDA device 0 among 2 EGL devices in total
+WindowlessContext: Unable to create windowless context
+```
+
+On Linux, this is often caused by an invalid driver or `libglvnd` installation. Follow the troubleshooting steps [below](#troubleshooting-tips). If the issue persists, feel free to file a Github issue.
+
+### Black squares on NVIDIA A100
+
+NVIDIA A100 GPUs may caused Habitat sensors to render black rectangular artifacts on some environments.
+
+If this manifests on your setup, update your CUDA drivers to a recent version (at least 12.2).
+
+## Graphics Troubleshooting Tips
+
+These steps aim to narrow down your graphics-related issues.
+
+#### General Tips
+
+1. Enable verbose logging.
+
+    Launch the application with the following environment variables: `MAGNUM_LOG=verbose` and `MAGNUM_GPU_VALIDATION=ON`. This will print additional information to the logs that often indicate the problem.
```

Comment:
Also you need to mention to include at least warning level habitat logging, I think, if not debug.